ui/apps/platform/src/hooks/useSearch.ts

```diff
@@ -1,27 +0,0 @@
-import { useLocation, useHistory } from 'react-router-dom';
```

Comment:
This was replaced with ui/apps/platform/src/hooks/useURLSearch.ts, which composes with the more general ui/apps/platform/src/hooks/useURLParameter.ts and provides the same API to the Risk Acceptance components.